x-pack/plugins/infra/public/containers/ml/api/ml_cleanup.ts

```diff
@@ -5,21 +5,25 @@
  */
 
 import * as rt from 'io-ts';
-import { pipe } from 'fp-ts/lib/pipeable';
-import { fold } from 'fp-ts/lib/Either';
-import { identity } from 'fp-ts/lib/function';
-import { npStart } from '../../../legacy_singletons';
+import type { HttpHandler } from 'src/core/public';
 
-import { getDatafeedId, getJobId } from '../../../../common/infra_ml';
-import { throwErrors, createPlainError } from '../../../../common/runtime_types';
+import { getDatafeedId, getJobId } from '../../../../common/log_analysis';
```

Comment:
This is wrong. Should be `import { getDatafeedId, getJobId } from '../../../../common/infra_ml';`

Reply:
Thanks, this is probably my mistake while resolving merge conflicts.